tests/simulator/test_initialization.py

```diff
@@ -502,7 +492,7 @@ def _test_density_decreases_as_1overSqrtN(self, dim, interp_order):
         plt.legend()
         plt.savefig("noise_nppc_minus_theory_interp_{}_{}.png".format(dim, interp_order))
         plt.close("all")
-        self.assertGreater(3e-2, noiseMinusTheory[1:].mean())
+        self.assertGreater(4e-2, noiseMinusTheory[1:].mean())
```

Comment:
is that still necessary (or unrelated to the weight /startIndex error?)

Reply:
I will recheck this